docs/user/alerting/alerting-production-considerations.asciidoc

```diff
@@ -33,3 +33,17 @@ For details on the settings that can influence the performance and throughput of
 === Deployment considerations
 
 {es} and {kib} instances use the system clock to determine the current time. To ensure schedules are triggered when expected, you should synchronize the clocks of all nodes in the cluster using a time service such as http://www.ntp.org/[Network Time Protocol].
+
+[float]
+[[event-log-ilm]]
+==== Event log index lifecycle managment
```

Comment:
Should this be a sibling to `Deployment considerations` instead of a sub-heading?

Reply:
Yeah, I think you're right.  Will change that.